src/components/Sections/SectionChart/SectionBarChart.js

```diff
@@ -159,18 +159,24 @@ const SectionBarChart = ({ data, style, dimensions, legend, chartProperties = {}
       return item;
     });
   }
+  let isFull = false;
+  if (!showOverflow && (preparedData.length * CHART_LEGEND_ITEM_HEIGHT > dimensions.height || stacked)) {
+    isFull = true;
+    dimensions.height = (preparedData.length * CHART_LEGEND_ITEM_HEIGHT) + 300;
```

Comment:
the 300 should be const

Reply:
whats the 300 for anyway?

Reply:
extra height of the chart after we calculate the room for the legend